### PR TITLE
Admin route

### PR DESCRIPTION
**Pull request by elmi-**
created server route for Admin

### CODE REVIEW

**Comment by ashokpeeta** [comment]
please use the titles for the routes appropriately

File: views/admin/list_orders.ejs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Home Page</title>
```
Comment (ashokpeeta) [inline views/admin/list_orders.ejs:5]: change the title accrodingly

File: views/admin/order_in_queue.ejs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Home Page</title>
```
Comment (ashokpeeta) [inline views/admin/order_in_queue.ejs]: please change the title accordingly

File: views/admin_menu.ejs [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Home Page</title>
```
Comment (ashokpeeta) [inline views/admin_menu.ejs]: please change the title accordingly

File: views/register.ejs [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Home Page</title>
```
Comment (ashokpeeta) [inline views/register.ejs]: please change the title accordingly

**Review by ashokpeeta — COMMENTED** [review]
Resolve the review comments

File: views/admin/list_orders.ejs (hunk 1 above)
Comment (anamshk) [reply views/admin/list_orders.ejs:5]: updated to the correct title page

File: views/admin/order_in_queue.ejs (hunk 2 above)
Comment (anamshk) [reply views/admin/order_in_queue.ejs]: updated to the correct title page

File: views/admin_menu.ejs (hunk 3 above)
Comment (anamshk) [reply views/admin_menu.ejs]: updated to the correct page

File: views/register.ejs (hunk 4 above)
Comment (anamshk) [reply views/register.ejs]: updated title

**Pull request merged**